packages/@aws-cdk/aws-amplify/lib/basic-auth.ts

```diff
@@ -91,7 +91,7 @@ export class BasicAuth {
     if (this.props.password) {
       return {
         ...config,
-        password: this.props.password.toString(),
+        password: this.props.password.unsafeUnwrap(), // Safe usage
```

Comment:
`unsafeUnwrap` is not about what's IN the secret. It's about where it's going.

It would be 

- unsafe: to put it into a `description` of `environmentVariables` field (because that will be readable later on).
- safe: to put it into a `password` field (because the service will presumably guard that field from being read later on).

If you use `unsafePlaintext()`, there's *nothing* we can do to make that safe: the secret will always appear in the template.

Reply:
Ah of course, thanks for the explanation!